pom.xml

```
@@ -120,8 +118,8 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.277.x</artifactId>
-                <version>909.vf028101f1bfa</version>
+                <artifactId>bom-2.289.x</artifactId>
+                <version>999999-SNAPSHOT</version>
```

version should be something like this:
```suggestion
                <version>10890.v0_33df40457</version>
```

---

```suggestion
                <version>1090.v0a_33df40457a_</version>
```